ninja/errors.py

```diff
@@ -28,6 +30,22 @@ class ConfigError(Exception):
     pass
 
 
+TModel = TypeVar("TModel", bound="ParamModel")
```

Comment:
This is a copy of `TModel` in `ninja.params.models`. I couldn't figure out a way to import it without causing cyclical import issues, because it's used below in `class ValidationErrorContext(Generic[TModel])`